src/content/docs/accounts/original-accounts-billing/original-users-roles/user-migration.mdx

```diff
@@ -58,33 +53,127 @@ If you'll need to do a good amount of partitioning of users' access to accounts
+The new user record that's created has the same login credentials: there is no need to reset passwords. If a user has a pending email verification status (pending being verified), that will also be transitioned over. 
 
+Important tips: 
 
-    ![A screenshot of what is shown when you have an email address associated with multiple New Relic logins](./images/login-multiple-accounts-view.png "Login UI for email addresses with multiple user records")
+* Ensure the new users' email addresses match their original user record email addresses, including matching exact case. We use email addresses as the key value to match users and, in a later step, to transition their user-associated assets. 
+* Once you complete this step and create new user records, we highly recommend completing the migration process without delay. If you don't complete the steps to migrate assets and delete the original user record, a user may see two user records when logging in (see image below) or else may be missing assets they expect to see (like dashboards). 
+
+ ![A screenshot of what is shown when you have an email address associated with multiple New Relic logins](./images/login-multiple-accounts-view.png "Login UI for email addresses with multiple user records")
 
     <figcaption>
-If your email address is associated with multiple logins, you can verify your email to see those accounts. In the example above, the “Original New Relic account” refers to that user record being on the older, original account/user model. 
+If a user with access to multiple organizations is not migrated for all organizations, they'll retain an "original" user record, in addition to the new record created. When logging in, they'll have a choice of which to log into. 
     </figcaption>
+
+
+## Step 6: Access settings [#access-settings]
+
+![User migration page 6](./images/user-migration-page-6.png "user-migration-page-6.png")
+
+This step allows you to determine the level of access their users have. If you need to map user access to specific accounts and roles, then you'll want to set up groups and access grants at this stage. 
+
+You'll need to create an access grant for each account that they want a group to have access to. For example, a DevOps group might have five different access grants, each giving the group access to a different account.  
+
+This is a procedure done separately from the user migration wizard procedure. For more on how to create access grants, see the [user management tutorial](/docs/accounts/accounts-billing/new-relic-one-user-management/tutorial-add-new-user-groups-roles-new-relic-one-user-model/#group-access).
```

Comment:
Maybe we can reword this to avoid the passive voice here...

Something like: "This procedure is done separately from the..."